cyences_app_for_splunk/bin/cs_product_list.py

```diff
@@ -320,28 +320,84 @@ def build_source_reviewer_search(by, values):
+                | tstats values(host) as hosts where index=* sourcetype="MSAD:NT6:Netlogon" by sourcetype index | rename sourcetype as sources | stats count values(index) as index dc(hosts) as host_count by sources | stats count values(*) as * | eval sourcetype=if(count>0,sourcetype,"MSAD:NT6:Netlogon")] | append [
+                | tstats values(host) as hosts where index=* sourcetype="MSAD:NT6:Replication" by sourcetype index | rename sourcetype as sources | stats count values(index) as index dc(hosts) as host_count by sources | stats count values(*) as * | eval sourcetype=if(count>0,sourcetype,"MSAD:NT6:Replication")] | append [
+                | tstats values(host) as hosts where index=* sourcetype="MSAD:NT6:Health" by sourcetype index | rename sourcetype as sources | stats count values(index) as index dc(hosts) as host_count by sources | stats count values(*) as * | eval sourcetype=if(count>0,sourcetype,"MSAD:NT6:Health")] | append [
+                | tstats values(host) as hosts where index=* sourcetype="MSAD:NT6:SiteInfo" by sourcetype index | rename sourcetype as sources | stats count values(index) as index dc(hosts) as host_count by sources | stats count values(*) as * | eval sourcetype=if(count>0,sourcetype,"MSAD:NT6:SiteInfo")] | append [
+                | tstats values(host) as hosts where index=* sourcetype="Script:TimesyncConfiguration" by sourcetype index | rename sourcetype as sources | stats count values(index) as index dc(hosts) as host_count by sources | stats count values(*) as * | eval sourcetype=if(count>0,sourcetype,"Script:TimesyncConfiguration")] | append [
+                | tstats values(host) as hosts where index=* sourcetype="Script:TimesyncStatus" by sourcetype index | rename sourcetype as sources | stats count values(index) as index dc(hosts) as host_count by sources | stats count values(*) as * | eval sourcetype=if(count>0,sourcetype,"Script:TimesyncStatus")] | append [
+                | tstats values(host) as hosts where index=* sourcetype="windows:certstore:ca:issued" by sourcetype index | rename sourcetype as sources | stats count values(index) as index dc(hosts) as host_count by sources | stats count values(*) as * | eval sourcetype=if(count>0,sourcetype,"windows:certstore:ca:issued")]""",
+                "earliest_time": "-1d@d",
+                "latest_time": "now",
+            }
+        ],
+    },
+    {
+        "name": "Windows DNS",
+        "metadata_count_search": '| tstats count where index=* source IN ("WinEventLog:DNS Server") OR sourcetype IN ("MSAD:NT6:DNS","MSAD:NT6:DNS-Health","MSAD:NT6:DNS-Zone-Information","PerfmonMk:DNS") ',
+        "macro_configurations": [
+            {
+                "macro_name": "cs_windows_idx",
+                "label": "Windows DNS Data",
+                "search": r"""| tstats count where index=* source="WinEventLog:DNS Server" by source | rename source as sources | stats count | eval sources=if(count>0, sources, "WinEventLog:DNS Server") | append [
```

Comment:
Can we simplify this into just two parts:
* source IN (xyz, abc) .... | append [sourcetype IN (a, b, c) ....]
Something like this?

Reply:
I kept it source/sourcetype wise because if there is 0 event then we have to disaplay.